src/composables/useDelegates.ts

```diff
@@ -24,8 +24,14 @@ type Governance = {
 const DELEGATES_LIMIT = 40;
 
 const DELEGATES_QUERY = gql`
-  query ($first: Int!, $skip: Int!) {
-    delegates(first: $first, skip: $skip, orderBy: delegatedVotes, orderDirection: desc) {
+  query ($first: Int!, $skip: Int!, $orderBy: Delegate_orderBy!, $orderDirection: OrderDirection!) {
+    delegates(
+      first: $first
+      skip: $skip
+      orderBy: $orderBy
+      orderDirection: $orderDirection
+      where: { tokenHoldersRepresentedAmount_gte: 0 }
```

Comment:
I think it's better this way:
```suggestion
      where: { tokenHoldersRepresentedAmount_gt: 0 }
```